crates/ruff_python_formatter/src/options.rs

```diff
@@ -349,3 +364,20 @@ where
         )),
     }
 }
+
+#[derive(CacheKey, Clone, Copy, Debug, PartialOrd, Ord, PartialEq, Eq, Default)]
+#[cfg_attr(
+    feature = "serde",
+    derive(serde::Serialize, serde::Deserialize),
+    serde(rename_all = "lowercase")
+)]
+#[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
+pub enum PythonVersion {
+    Py37,
+    #[default]
+    Py38,
```

Comment:
Should we add a comment stating that the default version should match with the `PythonVersion` in the `ruff_linter` crate?

Reply:
Great idea. I added a comment to both `PythonVersion`s